packages/blockchain-link-utils/src/stellar.ts

```diff
@@ -13,9 +13,15 @@ import {
 
 import type { TokenDetailByMint, Transaction } from '@trezor/blockchain-link-types';
 import { isCodesignBuild } from '@trezor/env-utils';
-import type { StellarAsset } from '@trezor/protobuf/src/messages';
 import { BigNumber } from '@trezor/utils/src/bigNumber';
 
+// copy paste from protobuf to avoid extra dependency
+export type StellarAsset = {
+    type: 0 | 1 | 2 | 'NATIVE' | 'ALPHANUM4' | 'ALPHANUM12';
+    code?: string;
+    issuer?: string;
+};
```

Comment:
copy pasting is not the best thing to do as well. maybe some deeper digging would help here..